config/config/wc-config.yaml

```
@@ -4,99 +4,187 @@ global:
   clusterName: ${CK8S_ENVIRONMENT_NAME}-wc
   baseDomain: "set-me"
   opsDomain: "set-me"
+
+  ## Default cert-manager issuer to use in ingresses
   issuer: letsencrypt-staging
+
+  ## Verify ingress certificates
+  ##
```

I think you can drop this and all other lines with only `##`. It will at least make the file a bit shorter.
Unless you have some reason for having them there.
```suggestion
  ## Verify ingress certificates
```